core/src/main/scala/org/http4s/Entity.scala

```diff
@@ -16,19 +16,58 @@
 
 package org.http4s
 
-import cats.Monoid
+import fs2.{Stream, Chunk}
+import cats._
 import cats.syntax.all._
 
-final case class Entity[+F[_]](body: EntityBody[F], length: Option[Long] = None)
+sealed trait Entity[F[_]]{
```

Comment:
Did covariance cause a problem?

Reply:
Yes. The Monoid disagreed very strongly with covariance.

Reply:
Do you remember what the issue was? I've tried making it covariant on a local branch and there wasn't a problem.

Reply:
Oh, I'm seeing the problem now - not sure why I didn't before.